package.json

```diff
@@ -34,5 +34,6 @@
     "production": [
       "IE 11 and last 2 versions"
     ]
-  }
+  },
+  "packageManager": "pnpm@10.10.0"
```

Comment:
What would happen if the user as the wrong pnpm installed globally?

Reply:
When the user executes `pnpm install`, pnpm will switch to the pnpm version specified by packageManager field.

Reply:
ref https://pnpm.io/settings#managepackagemanagerversions. https://github.com/pnpm/pnpm/pull/8363

Reply:
Don't we also need to change the pnpm action in the GitHub workflows then so that it automatically detect the pnpm version?

Reply:
Yes, please take a look at the documentation. https://github.com/pnpm/action-setup#inputs